src/radio/RadioGroup.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { RadioInitialState, RadioStateReturn, useRadioState } from "reakit";
+import { createContext } from "../utils";
+import { RadioCommonProps } from "./Radio";
+
+type RadioContextType = {
+  radioState?: RadioStateReturn;
+  radioSize?: RadioCommonProps["size"];
+};
+export const [RadioProvider, useRadioContext] = createContext<RadioContextType>(
```

Comment:
Export them separately below.

You remember our bug with Toast after building the component.
Bundlers have problem exporting the destructed array variables.

Reply:
Yeah yeah right.